test/index.js

```diff
@@ -2,12 +2,6 @@ var OSRM = require('../');
 var test = require('tape');
 var berlin_path = require('./osrm-data-path').data_path;
 
-test('constructor: throws if new keyword is not used', function(assert) {
```

Comment:
This is to capture errors for accidental instantiation. Imagine `var OSRM = require('osrm')();` instead of `var OSRM = require('osrm');`